File: config-biz/src/main/java/io/gitee/zicai/biz/mapper/AppPropsMapper.java
package io.gitee.zicai.biz.mapper;

import io.gitee.zicai.core.entity.AppProps;
import org.apache.ibatis.annotations.Param;
import tk.mybatis.mapper.common.Mapper;

import java.util.List;

public interface AppPropsMapper extends Mapper<AppProps> {

    List<AppProps> findProps(@Param("appName")String appName, @Param("appEnv")String appEnv);
}

File: config-biz/src/main/java/io/gitee/zicai/biz/mapper/AppPropsMapper.kt
package io.gitee.zicai.biz.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import io.gitee.zicai.core.entity.AppProps
import org.apache.ibatis.annotations.Param

interface AppPropsMapper : BaseMapper<AppProps> {

    fun findProps(@Param("appName") appName: String, @Param("appEnv") appEnv: String): List<AppProps>
}